ACM_General/payments/managers.py

```diff
@@ -78,6 +91,8 @@ def _create_transaction(self, stripe_token, **kwargs):
     def create_transaction(self, stripe_token, **kwargs):
         """
         Used to create a Transaction and save it to the database.
+        :param stripe_token: The stripe token.
```

Comment:
The stripe token associated with the transaction specified in the payment model.